infra/main.tf

```diff
@@ -99,11 +99,14 @@ module "feed-api" {
   source = "./feed-api"
 }
 
+
 module "functions-python" {
   source = "./functions-python"
   project_id  = var.project_id
   gcp_region  = var.gcp_region
   environment = var.environment
+  transitland_api_key = var.transitland_api_key
+  web_validator_url = var.validator_endpoint
```

Comment:
I don't see this variable used in the newly added code. What is the issue that is fixing?

Reply:
it's used [here](https://github.com/MobilityData/mobility-feed-api/blob/main/infra/functions-python/main.tf#L380) but it wasn't populated (it was using the default value across all envs). So I fixed as part of this PR

Reply:
I asked because I don't see the setting of this parameter in the GitHub actions.

Reply:
it's assigned here: https://github.com/MobilityData/mobility-feed-api/blob/main/.github/workflows/api-deployer.yml#L281-L286

Reply:
I see; renaming to validator_endpoint will definitely be easier to spot.

Reply:
done ✅